doc/source/getting-started/add-license-headers/faq.rst

```diff
@@ -0,0 +1,158 @@
+    to the hook in your ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --custom_copyright="custom copyright phrase"
+
+    This would change the copyright line to ``Copyright (C) 2025 custom copyright phrase.``
+
+.. dropdown:: How do you ignore checking for licensing information in the files?
+    :animate: fade-in-slide-down
+
+    To ignore checking for the MIT license in the files, add the ``--ignore_license_check``
+    argument to the hook in your ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --ignore_license_check
+
+.. dropdown:: How do you use a custom template?
+    :animate: fade-in-slide-down
+
+    To use a custom template, create the ``.reuse/templates/`` directory in the root of your
+    repository and add the jinja template to that directory. The custom template cannot be named
+    ``ansys.jinja2``, otherwise it will be removed after the hook is done running.
+
+    ::
+
+      project
+      ├── .reuse
+      │   └── templates
+      │       └── template_name.jinja2
+      ├── src
+      ├── examples
+      ├── tests
+      ├── .pre-commit-config.yaml
+      ├── pyproject.toml
+
+    Add the ``--custom_template`` argument to the hook in your ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --custom_template=template_name
+
+.. dropdown:: How do you use a custom license?
+    :animate: fade-in-slide-down
+
+    To use a custom license, create the ``LICENSES`` directory in the root of your
+    repository and add the license to that directory. The custom license cannot be named
+    ``MIT.txt``, otherwise it will be removed after the hook is done running.
+
+    ::
+
+      project
+      ├── LICENCES
+      │   └── license_name.txt
+      ├── src
+      ├── examples
+      ├── tests
+      ├── .pre-commit-config.yaml
+      ├── pyproject.toml
+
+    To use a custom license, add the ``--custom_license`` argument to the hook in your
+    ``.pre-commit-config.yaml`` file:
+
+    .. code:: yaml
+
+        - repo: https://github.com/ansys/pre-commit-hooks
+          rev: v0.5.1
+          hooks:
+          - id: add-license-headers
+            args:
+            - --custom_license=license_name
+
+    Licenses supported by ``REUSE`` can be found in the
+    `spdx/license-list-data <https://github.com/spdx/license-list-data/tree/main/text>`_ repository.
+    Please select a license text file from that repository, and copy it to the ``LICENSES`` directory.
```

Comment:
```suggestion
    Select a license text file from that repository and copy it to the ``LICENSES`` directory.
```
No comma is needed because the verbs share the same subject (the implied "you").